src/xrGame/ui/UIGameTutorial.h

```diff
@@ -67,6 +68,9 @@ class CUISequencer : public pureFrame, public pureRender, public IInputReceiver
         etsOverMainMenu = (1 << 6),
     };
     Flags32 m_flags;
+
+private:
+    pcstr m_name;
```

Comment:
It's better to place it after `m_sequencer_items`

Reply:
Updated